test/tests/functional/pbs_highreslog.py

```diff
@@ -148,16 +156,20 @@ def test_basic_tracejob(self):
         self.validate_trace_job_lines(jid=jid)
 
     def test_env_variable_overwrite(self):
+        """
+        Check env variable overwrites the pbs.conf value
+        """
+        a = {'PBS_LOG_HIGHRES_TIMESTAMP': 0}
+        self.du.set_pbs_config(confs=a, append=True)
         conf_path = self.du.parse_pbs_config()
         pbs_init = os.path.join(os.sep, conf_path['PBS_EXEC'],
                                 'libexec', 'pbs_init.d')
         cmd = ['sudo', 'PBS_LOG_HIGHRES_TIMESTAMP=1', pbs_init, 'restart']
-        self.du.run_cmd(cmd=cmd, as_script=True)
+        self.du.run_cmd(cmd=cmd, as_script=True, wait_on_script=True)
         j = Job(TEST_USER)
         jid = self.server.submit(j)
-        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid, offset=2)
```

Comment:
Just to give time to the job to go into running state. Its not completely necessary since expect will have multiple attempts, but to make more sure and have less number of expects I added this. I can remove it if you want.

Reply:
removed this